python/pysrc/kaskada/_ffi.pyi

```diff
@@ -44,6 +44,11 @@ class Expr:
     def session(self) -> Session: ...
     def execute(self, options: Optional[_ExecutionOptions] = None) -> Execution: ...
     def grouping(self) -> Optional[str]: ...
+    def plan(
+        self,
+        kind: Literal["initial_dfg", "final_dfg", "final_plan"],
```

Comment:
alternate naming could be `["initial_dfg", "optimized_dfg", "plan"]`?

Though..I wonder if we'd ever like to be able to apply optimizations one by one, and see the progression of the `dfg` for each step. Not sure what that would look like here; if we want that, we'd likely need to swap off the enum and literal.

Reply:
For the plan, I was thinking we'd eventually want to show `initial_plan` and `final_plan`. Definitely could see having more options here, although another way to do the progressino would be to allow the options to control which passes get run, and then view the `final_plan`.